### PR TITLE
Refresh governance document

### PR DESCRIPTION
**Pull request by sunxiaoguang**
Refresh the governance document as stated in #207

Closes: #207

### CODE REVIEW

**Review by zhangjinpeng87 — APPROVED** [review]
LGTM

File: GOVERNANCE.md [hunk 1]
```diff
@@ -75,10 +75,10 @@ The current list of Infra members is as below, in alphabetical order on last nam
 
 | Name                                  | GitHub                                           |
 | ------------------------------------- | :----------------------------------------------- |
-| Zili Chen <wander4096@gmail.com>      | [tisonkun](https://github.com/tisonkun)          |
+| Junchen Ye <yejunchen@pingcap.cn>     | [VelocityLight](https://github.com/VelocityLight)|
```
Comment (tisonkun) [inline GOVERNANCE.md:78]: Emmm .. I'm still actively oversee the org. Why remove without any communication or even notification?

File: GOVERNANCE.md [hunk 2]
```diff
@@ -75,10 +75,10 @@ The current list of Infra members is as below, in alphabetical order on last nam
 
 | Name                                  | GitHub                                           |
 | ------------------------------------- | :----------------------------------------------- |
-| Zili Chen <wander4096@gmail.com>      | [tisonkun](https://github.com/tisonkun)          |
```
Comment (tisonkun) [inline GOVERNANCE.md:78]: Emmm .. I'm still actively oversee the org. Why remove without any communication or even notification?

**Pull request merged**